screens/DaterProfile.tsx

```
@@ -0,0 +1,239 @@
+
+  useEffect(() => {
+    if (profile.matchRecommendations.length > 0) {
+      setShowRecommendations(true);
+    }
+
+    if (profile.friends.length > 0) {
+      setShowFriends(true);
+    }
+  }, [profile.matchRecommendations, profile.friends]);
+
+  return (
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@stylistic/js/indent)